.github/workflows/ci.yaml

```diff
@@ -148,7 +148,7 @@ jobs:
         run: ./project/scripts/sbt ";set ThisBuild/Build.scala2Library := Build.Scala2LibraryTasty ;scala3-bootstrapped/testCompilation i5; scala3-bootstrapped/testCompilation tests/run/typelevel-peano.scala; scala3-bootstrapped/testOnly dotty.tools.backend.jvm.DottyBytecodeTests" # only test a subset of test to avoid doubling the CI execution time
 
       - name: Test with Scala 2 library with CC TASTy (fast)
-        run: ./project/scripts/sbt "scala2-library-cc/compile; scala2-library-cc-tasty/compile" # TODO test all the test configurations in non-CC library (currently disabled due to bug while loading the library)
+        run: ./project/scripts/sbt "scala2-library-cc/compile; scala2-library-cc-tasty/compile; scala3-bootstrapped/testCompilation i3"
```

Comment:
What is `i3`? Is that a set of test files?

Reply:
All issues from `i3000` to `i3999` (`i300` to `i399`, ...) that have compilation tests. It is just a subset of tests that happens to not be contained in https://github.com/lampepfl/dotty/issues/19586#issuecomment-1925328488.

Reply:
Ah OK.